mailpoet/mailpoet
# Feat/let users design nicer confirmation emails - MAILPOET-4649

## Description

Let users design nicer confirmation emails.

This PR will add support for designing and customizing confirmation emails with the newsletter email editor.


## QA notes

* Visit the settings page : `/wp-admin/admin.php?page=mailpoet-settings#/signup`
* Click on the toggle "Enable visual subscription confirmation emails"
* Click on the "Save settings" button
* Click on "Open template editor."
* You may edit the template or not. Click on save.
* Close the editor
* Subscribe to a Form on the site.
* You should receive the confirmation email with the new HTML template


## Linked tickets

[MAILPOET-4649](https://mailpoet.atlassian.net/browse/MAILPOET-4649)
